app/controllers/topical_events_controller.rb

```diff
@@ -1,9 +1,18 @@
 class TopicalEventsController < ApplicationController
+  include PrometheusSupport
+
   enable_request_formats show: :atom
 
+  GRAPHQL_TRAFFIC_RATE = 0.01 # This is a decimal version of a percentage, so can be between 0 and 1
```

Comment:
I realise this was copied over for consistency within this code base. 
I was wondering if it would be better to make those strings to prepare to use env vars. Example: https://github.com/alphagov/govuk-helm-charts/blob/e99e8a384bbe86bc62e738a3c7778374dd898c10/charts/app-config/values-integration.yaml#L1394-L1395

Reply:
I'm going to raise another PR to switch this app to use env vars, so I'll switch to using strings in that PR.